lib/src/watch_event.dart

```diff
@@ -19,17 +19,38 @@ class WatchEvent {
 /// Enum for what kind of change has happened to a file.
 class ChangeType {
   /// A new file has been added.
-  static const ADD = ChangeType('add');
+  @Deprecated('Use add instead')
```

Comment:
I am generally of the opinion that these types of changes aren't worth the churn.

If @natebosch wants to override thats fine with me though.

Reply:
In general, agreed – but I also have this feeling we want to demonstrate best practices where we can. 

🤷

Reply:
@natebosch ?

Reply:
@jakemac53 does it change the balance of tradeoffs for you if we add fix information so that `dart fix` rewrites them for you?